lib/transformers/transformers.js

```diff
@@ -103,12 +103,23 @@ const mapOutReq = (incomingMessage, reqId, opts = {}) => {
 
   const metaHeaders = extractHeaders(headersRegex, incomingMessage.headers);
 
+  let metaBody = {};
+
+  if (method === 'POST' && opts.bodyKeys && incomingMessage.body) {
+    const jsonObject = JSON.parse(JSON.stringify(incomingMessage.body));
```

Comment:
Why are you doing this? Maybe toString would be ok.
What if request body is not a json.